docs/src/development/troubleshoot.md

```diff
@@ -175,7 +183,7 @@ If the result is still `running`, the build is stuck.
 In most regions, stuck builds terminate after one hour.
 In older regions (`us` and `eu`), create a [support ticket](https://console.platform.sh/-/users/~/tickets/open) to have the build killed.
 
-When a _deployment_ is blocked, you should try the following:
+When a *deployment* is blocked, you should try the following:
```

Comment:
Why this change? This actually goes against the Markdownlint rule I'd adopt (I'd go with underscore for emphasis and asterisks for bold).

Reply:
I was using the [standard markdownlint](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md049), which triggered here. What default settings are you using besides underscore for emphasis and asterisks for bold?

Reply:
I wouldn't personally make changes based on the standard settings without agreeing on the standard for here. :shrug:

Reply:
But I think it's not a big deal either way. Was just curious because I've used a different standard before.